app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,52 @@
+class ProjectsController < ApplicationController
+  before_action :set_project, only: [:show, :edit, :update, :destroy]
+
+  def index
+    @projects = Project.all
+  end
+
+  def show
+  end
+
+  def new
+    @project = Project.new
+  end
+
+  def create
+    @project = Project.new(project_params)
+    if @project.save
+      flash[:notice] = "Project has been created."
+      redirect_to @project
+    else
+      flash.now[:error] = "Project could not be saved."
+      render :new
+    end
+  end
+
+  def edit
+  end
+
+  def update
+    if @project.update_attributes(project_params)
+      redirect_to @project, notice: 'Project was sucessfully updated.'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.